README.md

```diff
@@ -1,70 +1,58 @@
-# Getting Started with Create React App
+Hello World UI
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app) and Redux.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+> This is an App that allows users to fetch a random greeting from a local hosted rails API [server](http://localhost:3000/api/greetings) built in this backend [project](https://github.com/Gambit142/hello-world-react-rails/tree/development/app). The greeting is displayed to the user through a dedicated route.
 
-In the project directory, you can run:
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- ### Major languages
+  - Javascript, CSS, and HTML
+- ### Frameworks
+  - ReactJS, Redux, React-Redux
+- ### Technologies used
+  - Webpack, Babelrc, Linters
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
 
-### `npm test`
+## Getting Started
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+To get a local copy up and running follow these simple example steps.
 
-### `npm run build`
+```
+git clone https://github.com/Gambit142/hello-world-react-ui.git
+```
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+and then run
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+```
+npm i
+npm start
+```
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Installation
 
-### `npm run eject`
+to build a production version to `bulid` folder run
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+```
+npm run build
+```
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Authors
```

Comment:
[OPTIONAL]

- Kindly change the header title Authors to Author, as this project is done solely. This will make your README appear more professional.

Reply:
Thank you for this suggestion, I have implemented it